packages/core/package.json

```diff
@@ -120,14 +120,14 @@
         "default": "./dist/test-utils/llm-mock.cjs"
       }
     },
-    "./agent/input-processor/processors": {
+    "./processors": {
       "import": {
-        "types": "./dist/agent/index.d.ts",
-        "default": "./dist/agent/index.js"
+        "types": "./dist/processors/index.d.ts",
+        "default": "./dist/processors/index.js"
       },
       "require": {
-        "types": "./dist/agent/index.d.ts",
-        "default": "./dist/agent/index.cjs"
```

Comment:
since we already released this we have to keep the original exports too

Reply:
true, we can add this to the breaking changes list